doc/classes/RenderingDevice.xml

```diff
@@ -2730,5 +2741,27 @@
 		<constant name="DRAW_IGNORE_ALL" value="720640" enum="DrawFlags" is_bitfield="true">
 			Ignore the previous contents of all attachments.
 		</constant>
+		<constant name="LATENCY_MODE_LOW_EXTREME" value="0" enum="LatencyMode">
+			Godot is willing to sacrifice a considerable amount of FPS (frames per second) to achieve the lowest possible latency.
+			It's generally recommended to use [constant LATENCY_MODE_LOW] instead, as the FPS cost tends to be too high.
+			It is strongly recommended this setting should only be set by the end user in user settings, and not be shipped by default.
```

Comment:
Avoid using newlines too often, as every newline creates a new paragraph. This is especially true for constants whose descriptions are designed to be more compact than properties/methods:

```suggestion
			The engine is willing to sacrifice a considerable amount of FPS (frames per second) to achieve the lowest possible latency. It's generally recommended to use [constant LATENCY_MODE_LOW] instead, as the FPS cost tends to be too high. It is strongly recommended this setting should only be set by the end user in user settings, and not be shipped by default.
```